test/unity/toolkits/drawing_classifier/test_drawing_classifier.cxx

```diff
@@ -199,26 +200,27 @@ BOOST_AUTO_TEST_CASE(test_drawing_classifier_init_training) {
   model.create_iterator_calls_.push_back(create_iterator_impl);
 
   model.init_model_calls_.emplace_back([=]() {
-
     std::unique_ptr<model_spec> nn_spec(new model_spec);
+    auto weight_init_fn = [](float* w, float* w_end) {
+      for (int i = 0; i < w_end - w; ++i) {
+        w[i] = static_cast<float>(i);
+      }
+    };
     nn_spec->add_convolution("test_layer", "test_input", 16, 16, 3, 3, 1, 1,
-                             model_spec::padding_type::SAME,
-                             /* weight_init_fn */ [](float* w, float* w_end) {
-                               for (int i = 0; i < w_end - w; ++i) {
-                                 w[i] = static_cast<float>(i);
-                               }
-                             });
+                             model_spec::padding_type::SAME, weight_init_fn);
     return nn_spec;
   });
 
+  /**
+   * TODO: const float_array_map& weights, const float_array_map&
+   * config. Until the nn_spec in C++ isn't ready, do not pass in any
```

Comment:
Having `config` be in a separate line makes this comment slightly less readable, I think.

Reply:
well, you will remove this later after you finish the TODO?.

Reply:
Yes, this TODO is partially updated in #2482